techniques/failures/F109.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html><html lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml">
+ 
+ <head><title>Failure of Success Criterion 3.3.8 and 3.3.9 due to forcing transcription of individual password characters</title>
+   
+ <link rel="stylesheet" type="text/css" href="../../css/sources.css" class="remove"></head><body><h1>Failure of Success Criterion 3.3.8 and 3.3.9 due to forcing transcription of individual password characters</h1><section class="meta"><p class="id">ID: F109</p><p class="technology">Technology: failures</p><p class="type">Type: Failure</p></section><section id="applicability"><h2>When to Use</h2>
+      <p>All technologies that require authentication.</p>
+   </section><section id="description"><h2>Description</h2>
+  <p>Requiring users to authenticate by entering a password in a different format from which it was originally created is a failure to meet Success Criteria 3.3.8 and 3.3.9 (unless alternative authenticaton methods are available). If a user is required to enter individual password characters across multiple fields, in a way that prevents pasting the password in a single action, it prevents use of a password manager or pasting from local copy of the password. This means users cannot avoid transcription, resulting in a <a href="../../understanding/22/accessible-authentication.html#dfn-cognitive-function-test">cognitive function test</a>. This applies irrespective of whether users are required to enter all characters in the password, or just a subset.</p>
+  
+  
+   </section><section id="examples"><h2>Examples</h2>
+  <p>These examples would prevent a user from entering a password in the same format in which the password was originally created:</p>
+     <ul>
+				<li>A fieldset that prompts a user to "Enter the 2nd, 6th and last characters of your password", with separate input fields for each character.</li>
+				<li>A fieldset that prompts a user to enter all digits of a 6-digit passcode, across 6 individual fields (unless the user can paste the entire 6-digit passcode into one field, and the remaining fields are populated automatically).</li>
```

Comment:
```suggestion
				<li>A fieldset that prompts a user to enter each digit of a passcode in separate inputs (unless the user can paste the entire passcode in the first input, and the remaining inputs are populated automatically).</li>
```

Reply:
Thanks @mbgower. I've committed your suggestion as the 'unless' part is already covered in the text above. I think it aids clarity to include it in the example here though - without out it, the example could potentially be compliant, as that  detail is left open.